lib/core/include/scipp/core/element/special_values.h

```diff
@@ -60,15 +56,15 @@ template <typename T> auto isneginf(T x) {
 } // namespace detail
 
 constexpr auto isposinf =
-    overloaded{special_value_args,
+    overloaded{special_value_args<>,
```

Comment:
Are this and `isneginf` not supported by Eigen?

Reply:
I don't think they make sense (unless you want to support them element-wise): You can have `sc.vector([INF, -INF, 0])`, what would this return?

Reply:
Good point. Leave them out